source/blood/src/sfx.cpp

```diff
@@ -396,11 +396,14 @@ void sfxKill3DSound(spritetype *pSprite, int a2, int a3)
     }
 }
 
-void sfxKillAllSounds(void)
+void sfxKillAllSounds(spritetype *pSprite)
```

Comment:
Make separate variant for sprites

Reply:
I'm not sure if I should refactor the duplicated code into a separate method or it is not advised in this codebase because the method will be called inside a loop.